packages/itwin/tree-widget/api/tree-widget-react.api.md

```diff
@@ -451,7 +453,7 @@ export interface TreeContextMenuProps {
 export interface TreeDefinition {
     getLabel: () => string;
     id: string;
-    render: () => React.ReactNode;
+    render: (props: TreeRenderProps) => React.ReactNode;
```

Comment:
@saskliutas @jasdom, looks like we missed a breaking change. Can props be made optional?

Reply:
I don't think this is a breaking change. These props are always provided by `TreeSelector` when rendering selected tree. https://github.com/iTwin/viewer-components-react/blob/9d296a06dd2e2290364692e02b109e9ac591a5e9/packages/itwin/tree-widget/src/components/TreeSelector.tsx#L64

This callback should never be called by consumer. And consumers can keep passing render callback that does not need props:
```ts
 {
  id: CategoriesTreeComponent.id,
  getLabel: CategoriesTreeComponent.getLabel,
  render: () => <CategoriesTreeComponent />,
},
```

Reply:
Agreed.